src/servers/volume/server/web-api.ts

```diff
@@ -206,4 +211,33 @@ function queryDone() {
     if (State.shutdownOnZeroPending) {
         process.exit(0);
     }
+}
+
+async function healthCheck(res: express.Response) {
+    if (ServerConfig.healthCheckPath?.length === 0) {
+        healthCheckResponse(res, true);
+        return;
+    }
+
+    for (const path of ServerConfig.healthCheckPath) {
+        try {
+            // assert readable file
+            await promises.access(path, constants.R_OK);
+        } catch (e) {
+            ConsoleLogger.error(`Error accessing path ${path}:`, e);
+            healthCheckResponse(res, false, 'Failed to access data from file system.');
+            return;
+        }
+    }
+    healthCheckResponse(res, true);
+}
+
+function healthCheckResponse(res: express.Response, success: boolean, msg?: string) {
+    res.writeHead(success ? 200 : 500, {
+        'Content-Type': 'text/plain',
```

Comment:
Should this be `application/json` instead?

Reply:
`text/plain` tends to be easier to work with, at least for the monitoring tools that RCSB PDB uses (that match with an expected string indicating a healthy status).

Could wrap everything in an object and deliver as JSON. A simple string and `text/plain` should be more appropriate for our use-case.